README.md

```diff
@@ -1,18 +1,18 @@
 ### What is Multinode?
 
-**Multinode** allows you to rapidly deploy scalable applications with **asynchronous tasks**.
+**Multinode** allows you to rapidly deploy applications that run **asynchronous tasks** scalably on the cloud.
```

Comment:
- "applications with async tasks" doesn't read right. Should be "applications that run async tasks"
- The original version omitted a crucial message: we're scaling these tasks _on the cloud_